blocks/library/pullquote/index.js

```diff
@@ -80,7 +80,7 @@ registerBlockType( 'core/pullquote', {
 			<blockquote key="quote" className={ className }>
 				<Editable
 					multiline="p"
-					value={ toEditableValue( value ) }
+					value={ value && toEditableValue( value ) }
```

Comment:
Could we make `toEditableValue` handle the empty case? Thinking in case we introduce another usage of it. Would be a simple change to use Lodash's variant, which handles empty values:

```js
const toEditableValue = value => map( value, ( subValue => subValue.children ) );
```

Reply:
@aduth Thanks for the suggestion. I'll try that approach. How do I proceed now since it was merged and I deleted the branch? Do I just restore the branch and push again?

Reply:
It'd be fine to create a new separate branch with the proposed change.